erddap/templates/private/dataset.deployment.xml

```diff
@@ -301,184 +301,35 @@
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>time_uv_qc</sourceName>
-        <destinationName>time_uv_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">time_uv Variable Quality Flag</att>
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>lat_uv_qc</sourceName>
-        <destinationName>lat_uv_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">lat_uv Variable Quality Flag</att>
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>lon_uv_qc</sourceName>
-        <destinationName>lon_uv_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">lon_uv Variable Quality Flag</att>
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>u_qc</sourceName>
-        <destinationName>u_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">u Variable Quality Flag</att>
-        </addAttributes>
-    </dataVariable>
-
-    <dataVariable>
-        <sourceName>v_qc</sourceName>
-        <destinationName>v_qc</destinationName>
-        <dataType>byte</dataType>
-        <addAttributes>
-            <att name="ioos_category">Quality</att>
-            <att name="long_name">v Variable Quality Flag</att>
+            <att name="long_name">{{dest_var_remaps.get(req_var, req_var).replace('_qc', '')}} Variable Quality Flag</att>
+            {% if '_FillValue' not in qc_var_types['gen_qc'][req_var] %}
+            <att name='_FillValue' type='byte'>-127</att>
+            {% endif %}
         </addAttributes>
     </dataVariable>
+    {% endfor %}
 
-    {% if include_qartod %}
-    {% for var in qartod_variables %}
+    {# if there are QARTOD variables defined, populate them #}
+    {% if qc_var_types['qartod'] | length > 0 %}
+    {% for qartod_var in (qc_var_types['qartod']) %}
     <dataVariable>
-        <sourceName>{{var}}</sourceName>
-        <destinationName>{{var}}</destinationName>
+        <sourceName>{{qartod_var}}</sourceName>
+        <destinationName>{{qartod_var}}</destinationName>
         <dataType>byte</dataType>
         <addAttributes>
             <att name="ioos_category">Quality</att>
+            {# all qartod variables should have _FillValue, missing_value,
+               flag_values, and flag_meanings defined #}
+           <att name="flag_values">1B, 2B, 3B, 4B, 9B</att>
+           <att name="flag_meanings">GOOD NOT_EVALUATED SUSPECT BAD MISSING</att>
+           <att name="_FillValue" type='byte'>2</att>
```

Comment:
@kerfoot what's your opinion on setting the _FillValue to a 2 (NOT_EVALUATED) flag?